publish/dir.props

```diff
@@ -49,6 +49,5 @@
     <PublishRid Include="win-arm64" />
     <PublishRid Include="linux-arm" />
     <PublishRid Include="rhel.7-x64" />
```

Comment:
@gkhanna79 @eerhardt @ericstj  Should we be removing all the non-portable RID's from this list?